app/helpers/application_helper/button/ems_admin_ui.rb

```diff
@@ -0,0 +1,5 @@
+class ApplicationHelper::Button::EmsAdminUI < ApplicationHelper::Button::Basic
```

Comment:
Youn don't need a new class like this. Use this one: `app/helpers/application_helper/button/generic_feature_button.rb`

Reply:
Thanks, I'll use `GenericFeatureButton` as suggested.